semgrep/semgrep/core_runner.py

```diff
@@ -272,6 +261,80 @@ def get_files_for_language(
             ) from ex
         return list(targets)
 
+    def _get_targets(
+        self, rules: List[Rule], target_manager: TargetManager, all_targets: Set[Path]
+    ) -> List[Dict[str, Any]]:
+        target_info: Dict[Tuple[Path, Language], List[RuleId]] = {}
+
+        for rule in rules:
+            for language in rule.languages:
+                targets = self.get_files_for_language(language, rule, target_manager)
+
+                for target in targets:
+                    all_targets.add(target)
+                    t = (target, language)
+                    if t in target_info:
+                        target_info[t].append(RuleId(rule.orig_id))
```

Comment:
idk if it's safe to use orig_id when elsewhere in the code we consistently use id?